Makefile

```diff
@@ -190,11 +190,17 @@ crds: $(CONTROLLER_GEN) $(YQ)
 	@echo Updating CRD manifests
 	@build/crds/build-crds.sh $(CONTROLLER_GEN) $(YQ)
 
-gen-rbac: $(HELM) $(YQ) ## generate RBAC from Helm charts
+gen-rbac: $(HELM) $(YQ) ## Generate RBAC from Helm charts
 	@# output only stdout to the file; stderr for debugging should keep going to stderr
 	HELM=$(HELM) ./build/rbac/gen-common.sh
 	HELM=$(HELM) ./build/rbac/gen-nfs-rbac.sh
 
+docs-preview: ## Preview the documentation through mkdocs
+	mkdocs serve
+
+docs-build:  ## Build the documentation to the `site/` directory
+	mkdocs build --strict
```

Comment:
This will fail the docs build if there is a broken link? Could you push a break temporarily so we can see the new action fail? Or did you already do that?

Reply:
Works as intended :-)

![Ci docs-check fails succesfully](https://user-images.githubusercontent.com/3718398/176034509-9d3d1212-2d31-4a7c-8b54-58a8418cbbae.png)

Reply:
Looks perfect!